packages/server/src/modules/app/index.ts

```diff
@@ -82,7 +83,11 @@ application.start = async () => {
 };
 
 application.test.start = async () => {
-  connect();
+  generateUniqueDbName();
+  await connect();
   // TODO - should be a part of the lifecycle hook/event
   app.use(application.middleware.errorHandler);
 };
+
+const generateUniqueDbName = () =>
```

Comment:
`utils.ts` file in the same module is a better place for the function definition. Ideally trying to avoid index file pollution.

Reply:
Sure. This is at first step of "Make It Work, Make It Right, Make It Fast"